### PR TITLE
Move to MAX-Framework

### PR DESCRIPTION
**Pull request by MLnick**
Move this model to [MAX-Framework](https://github.com/IBM/MAX-Framework).

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,2 +1,3 @@
 h5py
-tensorflow==1.9.0
\ No newline at end of file
+tensorflow==1.9.0
+-e git+https://github.com/IBM/MAX-Framework.git#egg=maxfw
```
[the remark this answers is not in the document]
Comment (ajbozarth) [reply requirements.txt]: We should cut releases of MAX-Framework according to the same release rules as models and tie a model release to a framework release

**Comment by ajbozarth** [comment]
Other than my comment about versioning both the framework and any model update to use it, this LGTM (from a code side, I didn't test the model)

File: requirements.txt (hunk 1 above)
[the remark this answers is not in the document]
Comment (djalova) [reply requirements.txt]: I tagged the current version on the repo as v0.1
https://github.com/IBM/MAX-Framework/tree/v0.1

[the remark this answers is not in the document]
Comment (MLnick) [reply requirements.txt]: @bdwyer2 yes the idea would be for a version of `maxfw` to be tied to a version of the base docker image (using tags on the MAX-Base repo). And as @ajbozarth says we should install a specific version tag. 

Once it is up on pip we can also do a pip install of the specific version.

This is a placeholder (WIP) effort to illustrate that it does work.

**Comment by MLnick** [comment]
since `max-base` now has `maxfw`, I removed the pip install from this model `requirements.txt`.

Re-tested and all seems to work. PR can be reviewed again @ajbozarth @djalova @bdwyer2

**Pull request merged**